docs/resources/rds_pg_hba.md

```diff
@@ -0,0 +1,75 @@
+---
+subcategory: "Relational Database Service (RDS)"
+---
+
+# huaweicloud_rds_pg_hba
+
+Manages an RDS PostgreSQL hba resource within HuaweiCloud.
```

Comment:
```suggestion
Manages RDS PostgreSQL hba resource within HuaweiCloud.
```
By the way, `hba` should be an abbreviated noun, should it be capitalized?

Reply:
it is not needed, the RDS service itself uses the abbreviation
it is more appropriate to add the world of an, because this resource can only manage one instance